Week 4/Unit 07/Assignment 1/lab/src/main/java/fpt/lab/config/AppConfig.java

```diff
@@ -0,0 +1,19 @@
+package fpt.lab.config;
+
+import org.springframework.context.annotation.Bean;
+
+import fpt.lab.controller.EmployeeWork;
+import fpt.lab.model.Employee;
+
+public class AppConfig {
+    
+    @Bean
+    public EmployeeWork employeeWork() {
```

Comment:
Can we inject employeeWork to employee bean via param ?

Reply:
Yes, we can. By defining the employee bean method to accept an `EmployeeWork` parameter, Spring will resolve and inject the appropriate `EmployeeWork` bean instance when creating the `Employee` bean. This ensures that the `Employee` bean is correctly wired with its dependencies.